### PR TITLE
AISDK-108: Add delete endpoint for custom vocabulary

### PR DESCRIPTION
**Pull request by k-weng**

### CODE REVIEW

File: tests/test_customvocabulary.py [hunk 1]
```diff
@@ -1,30 +1,72 @@
...
+from src.rev_ai.custom_vocabularies_client import RevAiCustomVocabulariesClient
 from src.rev_ai.models import CustomVocabulary
 
-phrases = ["Patrick Henry Winston", "Noam Chomsky"]
+try:
+    from urllib.parse import urljoin
+except ImportError:
+    from urlparse import urljoin
+
+TOKEN = "token"
```
Comment (amikofalvy) [inline tests/test_customvocabulary.py:11]:
```suggestion
ACCESS_TOKEN = "your_access_token"
```

File: tests/test_customvocabulary.py [hunk 2]
```diff
@@ -1,30 +1,72 @@
...
+CV_ID = "cv1"
```
Comment (amikofalvy) [inline tests/test_customvocabulary.py:12]:
```suggestion
CV_ID = "cv12345678"
```

**Review by amikofalvy — APPROVED** [review]

File: examples/custom_vocabularies_example.py [hunk 3]
```diff
@@ -41,11 +41,13 @@
...
     elif status == "failed":
         print("Job Failed : {}".format(custom_vocabularies["failure_detail"]))
         break
 
     if status == "complete":
         print("SUCCESS: {}".format(custom_vocabularies))
         break
+
+print("Deleting custom vocabulary")
+client.delete_custom_vocabulary(job_id)
```
Comment (amikofalvy) [inline examples/custom_vocabularies_example.py]: I would comment out this line and the previous.For someone who just runs the example of application and doesn't read every single line especially the last line in the file they would not understand why their custom vocabulary doesn't exist.

Comment (amikofalvy) [reply examples/custom_vocabularies_example.py]: To be clear I'd like to keep this line to illustrate how to use the delete endpoint. I just don't want it to execute

**Review by amikofalvy — APPROVED** [review]

File: examples/custom_vocabularies_example.py (hunk 3 above)
Comment (k-weng) [reply examples/custom_vocabularies_example.py]: Makes sense!

**Pull request merged**